horreum-integration-tests/src/test/java/io/hyperfoil/tools/horreum/it/HorreumClientIT.java

```diff
@@ -129,6 +130,51 @@ public void testJavascriptExecution() throws InterruptedException {
         Assertions.assertEquals("foobar", preview.value.textValue());
     }
 
+    @org.junit.jupiter.api.Test
+    public void testJavascriptExecutionWithArray() throws InterruptedException {
```

Comment:
Added new IT to ensure the Javascript invocation does work with array objects, the same test fails if run against `master`.